LongReadRNABenchmark/LongReadRNABenchmark.wdl

```diff
@@ -0,0 +1,407 @@
+    }
+
+    call LongReadRNABenchmarkTasks.DenovoAnalysis as DenovoAnalysisFlames {
+        input:
+            toolName = "flames",
+            gtfList = [IsoQuant.isoQuantGTF, StringTie.stringTieGTF, Bambu.bambuGTF, Flair.flairGTF, Talon.talonGTF, Flames.flamesGFF]
+    }
+
+    call LongReadRNABenchmarkTasks.DenovoStats as DenovoStatsIsoQuant {
+        input:
+            trackingFile = DenovoAnalysisIsoQuant.tracking,
+            toolName = "isoquant",
+            numTools = 6
+    }
+
+    call LongReadRNABenchmarkTasks.DenovoStats as DenovoStatsStringTie {
+        input:
+            trackingFile = DenovoAnalysisStringTie.tracking,
+            toolName = "stringtie",
+            numTools = 6
+    }
+
+    call LongReadRNABenchmarkTasks.DenovoStats as DenovoStatsBambu {
+        input:
+            trackingFile = DenovoAnalysisBambu.tracking,
+            toolName = "bambu",
+            numTools = 6
+    }
+
+    call LongReadRNABenchmarkTasks.DenovoStats as DenovoStatsFlair {
+        input:
+            trackingFile = DenovoAnalysisFlair.tracking,
+            toolName = "flair",
+            numTools = 6
+    }
+
+    call LongReadRNABenchmarkTasks.DenovoStats as DenovoStatsTalon {
+        input:
+            trackingFile = DenovoAnalysisTalon.tracking,
+            toolName = "talon",
+            numTools = 6
+    }
+
+    call LongReadRNABenchmarkTasks.DenovoStats as DenovoStatsFlames {
+        input:
+            trackingFile = DenovoAnalysisFlames.tracking,
+            toolName = "flames",
+            numTools = 6
+    }
+
+    call LongReadRNABenchmarkTasks.SummarizeAnalysis as SummarizeAnalysisReduced {
+        input:
+            inputList = [ReducedAnnotationAnalysisIsoQuant.novel, ReducedAnnotationAnalysisStringTie.novel, ReducedAnnotationAnalysisBambu.novel ,ReducedAnnotationAnalysisFlair.novel ,ReducedAnnotationAnalysisTalon.novel, ReducedAnnotationAnalysisFlames.novel],
+            toolNames = ["isoquant" , "stringtie", "bambu", "flair", "talon", "flames"],
+            datasetName = datasetName,
+            analysisType = "reduced"
+    }
+
+    call LongReadRNABenchmarkTasks.SummarizeAnalysis as SummarizeAnalysisReffree {
+        input:
+            inputList = [ReferenceFreeAnalysisIsoQuant.stats, ReferenceFreeAnalysisStringTie.stats, ReferenceFreeAnalysisIsoSeq.stats ,ReferenceFreeAnalysisCupcake.stats],
+            toolNames = ["isoquant" , "stringtie", "isoseq", "cupcake"],
+            datasetName = datasetName,
+            analysisType = "reffree"
+    }
+
+    call LongReadRNABenchmarkTasks.PlotAnalysisSummary as PlotAnalysisSummaryReduced {
+        input:
+            summary = SummarizeAnalysisReduced.summary,
+            datasetName = datasetName,
+            analysisType = "reduced"
+    }
+
+    call LongReadRNABenchmarkTasks.PlotAnalysisSummary as PlotAnalysisSummaryReffree {
```

Comment:
Is it worth also including a small task to make bar plots of the denovo stats?

Reply:
Added new task for this. Resolved.